build.sbt

```diff
@@ -8,21 +8,22 @@ lazy val `play-mailer` = (project in file("."))
     scalaVersion := scala212,
     crossScalaVersions := Seq(scala211, scala212)
   )
-    
+
 val PlayVersion = playVersion("2.6.0-RC2")
 
 libraryDependencies ++= Seq(
-  "com.typesafe.play" %% "play" % PlayVersion % Provided,
+  "com.google.inject" % "guice" % "4.0", // 4.0 to maybe make it work with 2.5 and 2.6
```

Comment:
This should work for both since for Play 2.6 this version will be evicted. But it would be good to run the tests against both versions of Play here (to ensure that changes made here will work as expected). You can configure Travis to run commands for multiple versions and read `PlayVersion` from some parameter then.

What do you think?